addons/weather/functions/fnc_updateServerWeather.sqf

```diff
@@ -9,12 +9,16 @@
  * None
  *
  * Example:
- * [] call ace_weather_fnc_updateAceWeather
+ * [] call ace_weather_fnc_updateServerWeather
  *
  * Public: No
  */
 #include "script_component.hpp"
 
+if (!GVAR(enabled)) exitWith {
+    missionNamespace setVariable [QGVAR(currentOvercast), 0, true];
```

Comment:
could just do the exitWith in XEH_postServerInit.sqf/ace_settingsInitialized
no point in running a PFEH to set a static value

Reply:
One of the design goals was to be able to switch the weather module on and off during the mission.

Reply:
We could simplify the code in many places, if we drop that objective.

Reply:
> One of the design goals was to be able to switch the weather module on and off during the mission.

y tho?

Reply:
`GVAR(currentOvercast)` needs to be set to `0` on all clients, when the Weather module is disabled.

Reply:
no, i mean, why  should anyone want to turn it off mid mission

Reply:
Ok, let's get rid of it.